client/src/pages/Home.js

```diff
@@ -42,14 +43,14 @@ function Home() {
           <button onClick={() => hideForm()}>Cancel</button>
           <input type="submit" disabled={!name} value="Create list" />
         </form>
-        <button onClick={() => displayForm()}>Add List</button>
         <List>
           {error && <div>Could not get data. Please cry.</div>}
           {loading && <div>Loading...</div>}
           {lists?.map((list) => (
             <ListItem key={list.id} list={list} href={`/home/${list.id}`} />
           ))}
         </List>
+        <FAB displayForm={() => displayForm()} />
```

Comment:
I would rename the displayForm-prop into onClick or handleClick, because that's the functionality you are defining here :) 
Your component would call the onClick / handleClick function when clicked.

Reply:
I know. This was just to check, if it's working here. I will rename this because and implement a useState here